modules/bclconvert/main.nf

```diff
@@ -0,0 +1,84 @@
+process BCLCONVERT {
+    tag 'demultiplexing'
```

Comment:
This might want to be 
```suggestion
    tag '$samplesheet'
```
To support multiple runs. Otherwise, they'll all be tagged demultiplexing.